src/RA_Dlg_AchEditor.cpp

```diff
@@ -982,9 +1001,20 @@ INT_PTR Dlg_AchievementEditor::AchievementEditorProc(HWND hDlg, UINT uMsg, WPARA
                             if (pActiveAch == nullptr)
                                 return FALSE;
 
-                            TCHAR buffer[16];
-                            if (GetDlgItemText(hDlg, IDC_RA_ACH_POINTS, buffer, 16))
+                            ra::tstring buffer(16, TCHAR());
+                            if (GetDlgItemText(hDlg, IDC_RA_ACH_POINTS, buffer.data(), 16))
                             {
+                                // TODO: Put some error messages in message boxes to tell users what happend
```

Comment:
TODO: address this in a separate PR. It's hard enough finding the changes associated to the description with all the formatting changes.

Reply:
I'll turn off auto formatting and just format by selection of changes code